modules/process/pages/generate-pdf-from-an-office-document.adoc

```diff
@@ -38,6 +38,16 @@ Rendering issues (at worst the element will not be visible in the generated PDF;
 * HTML conversion generates a single html file. For that matter, linked resources (like images, ...) are not supported. If you have images, use PDF conversion instead. +
 This connector is build upon *XdocReport* third party library, for more information check their Wiki at https://github.com/opensagres/xdocreport/wiki
 
-== Troubleshooting
-
-You may encounter memory issues using this connector as it has substantial size of dependencies. To workaround those issues you may increase the maximum memory allocation value of the Tomcat server JVM. Go to Preferences \-> Server settings \-> Increase the Tomcat Maximum memory allocation (`-Xmx` JVM argument) value to *1024* (512 by default). Do not forget to use the same value when configuring your production Bundle.
+== image:ROOT/assets/images/images/troubleshooting.png[troubleshooting-icon] Troubleshooting
+
+*Symptom*: +
+Users may encounter delays (long response times) when using this connector.
+*Cause:* +
+Memory issue as it has substantial size of dependencies. 
+*Potential solution:* +
+You may increase the maximum memory allocation value of the Tomcat server JVM. +
+To do so:
+ * Go to Preferences \-> Server settings \
```

Comment:
Here you can you the `menu` macro: https://docs.antora.org/antora/2.3/asciidoc/ui-macros/#set-the-experimental-attribute
You can check https://github.com/bonitasoft/bonita-doc/pull/1826/files for a quick how-to (including how to declare the `experimental` AsciiDoc attribute within the page)

Reply:
Thanks. I'm not sure about the added value compared to just write the bread crumbs by hand.
But I'm curious, and ready to try.